signifyd/signifyd.module

```diff
@@ -54,4 +54,58 @@ class SignifydApi {
 
     return $resp;
   }
+  
+  /**
+	* Get a case details from Signifyd.
+	*/
+	static function get_case($id) {
+		$api_key  = variable_get('signifyd_api_key', '');
+		$api_base = variable_get('signifyd_api_url', '');
+		$api_url  = $api_base . '/cases/'.$id;
+		
+		$resp = drupal_http_request($api_url, array(
+			'method' => 'GET',
+			'headers' => array(
+				'Content-Type' => 'application/json',
+				'Authorization' => 'Basic ' . base64_encode($api_key),
+			),
+		 ));
+		
+		watchdog('signifyd', 'API Get Response: <pre>'.print_r($resp, true).'</pre>');
+		
+		return $resp;
+	}
+	
+	/**
+	* Get a guarentee canceled and refunded
+	*/
+	static function cancel_guarentee($id) {
```

Comment:
thinking lets rename to 'cancel_case_guarantee'
update function comments so its clear "this function stops a case from being reviewed for guarantees" - or smth like that.

Reply:
i'm open here, i just think the naming should be more clear

Reply:
Good ideas worth updating. I've made them both more clear.